packages/http-client-csharp/emitter/src/type/type-cache.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { SdkType } from "@azure-tools/typespec-client-generator-core";
+import { InputEnumType, InputModelType, InputType } from "./input-type.js";
+
+export interface TypeCache {
```

Comment:
Maybe we can change it to `SdkInputTypes` to indicate to include all the types. Personally I don't like `Cache` in the interface name.

Reply:
I changed it to `SdkTypeMap`, how about this name?

Reply:
ok.  Maybe `SdkTypeMaps` ?

Reply:
I do not think this represent multiple maps. The map of models and enums are "views" of the first map.